ui/components/ui/export-text-container/export-text-container.component.js

```diff
@@ -1,51 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Copy from '../icon/copy-icon.component';
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
-import { exportAsFile } from '../../../helpers/utils/export-utils';
+import { ButtonSecondary } from '../../component-library';
 
-function ExportTextContainer({
-  text = '',
-  onClickCopy = null,
-  onClickDownload = null,
-}) {
+function ExportTextContainer({ text = '', onClickCopy = null }) {
+  const ONE_MINUTE = 1000 * 60;
   const t = useI18nContext();
-  const [copied, handleCopy] = useCopyToClipboard();
+  const [copied, handleCopy] = useCopyToClipboard(ONE_MINUTE);
 
   return (
     <div className="export-text-container">
       <div className="export-text-container__text-container">
         <div className="export-text-container__text notranslate">{text}</div>
       </div>
-      <div className="export-text-container__buttons-container">
+      <div className="export-text-container__button">
         <div
-          className="export-text-container__button export-text-container__button--copy"
+          className="export-text-container__button export-text-container__button--copy-only"
           onClick={() => {
             if (onClickCopy) {
               onClickCopy();
             }
             handleCopy(text);
           }}
         >
-          <Copy size={17} color="var(--color-primary-default)" />
-          <div className="export-text-container__button-text">
+          <ButtonSecondary className="export-text-container__button">
             {copied ? t('copiedExclamation') : t('copyToClipboard')}
-          </div>
-        </div>
-        <div
-          className="export-text-container__button"
-          onClick={() => {
-            if (onClickDownload) {
-              onClickDownload();
-            }
-            exportAsFile('', text);
-          }}
-        >
-          <img src="images/download.svg" alt="" />
-          <div className="export-text-container__button-text">
-            {t('saveAsCsvFile')}
-          </div>
+          </ButtonSecondary>
         </div>
       </div>
     </div>
```

Comment:
Using the `Box` component and some design system component here could eliminate the need for any scss here!

Imports: 
```
import Box from '../box';
import { Button, BUTTON_TYPES, Text } from '../../component-library';
import {
  ALIGN_ITEMS,
  BORDER_RADIUS,
  COLORS,
  DISPLAY,
  FLEX_DIRECTION,
  JUSTIFY_CONTENT,
  TEXT,
} from '../../../helpers/constants/design-system';
```

JSX
```jsx
  return (
    <Box
      className="export-text-container"
      display={DISPLAY.FLEX}
      justifyContent={JUSTIFY_CONTENT.CENTER}
      flexDirection={FLEX_DIRECTION.COLUMN}
      alignItems={ALIGN_ITEMS.CENTER}
      borderRadius={BORDER_RADIUS.MD}
      borderColor={COLORS.BORDER_DEFAULT}
      padding={4}
    >
      <Text variant={TEXT.BODY_LG_MEDIUM} marginBottom={4}>
        {text}
      </Text>
      <Button
        onClick={() => {
          if (onClickCopy) {
            onClickCopy();
          }
          handleCopy(text);
        }}
        type={BUTTON_TYPES.SECONDARY}
      >
        {copied ? t('copiedExclamation') : t('copyToClipboard')}
      </Button>
    </Box>
  );
```

This is just a question but is there any benefit to using a named function over an arrow function for the onClick handler?

Reply:
no, in this case there isn't any difference